modules/nodes-scheduler-taints-tolerations-projects.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-scheduler-taints-tolerations.adoc
+// * post_installation_configuration/node-tasks.adoc
+
+[id="nodes-scheduler-taints-tolerations-projects_{context}"]
+= Creating a project with a node selector and toleration
+
+You can create a project that uses a node selector and toleration to control the placement of pods onto specific nodes. Any subsequent resources created in the project are then scheduled on nodes that have a taint matching the toleration.
+
+.Prerequisites
+
+* A label for node selection has been added to one or more nodes by using a machine set or editing the node directly.
+* A taint has been added to one or more nodes by using a machine set or editing the node directly.
+
+.Procedure
+
+. Create a `Project` resource definition, specifying a node selector and toleration in the `metadata.annotations` section:
++
+.Example `project.yaml` file
+[source,yaml]
+----
+kind: Project
+apiVersion: project.openshift.io/v1
+metadata:
+  name: <project_name> <1>
+  annotations:
+    openshift.io/node-selector: '<label>' <2>
+    scheduler.alpha.kubernetes.io/defaultTolerations: >-
+      [{"operator": "Exists", "effect": "NoSchedule", "key":
+      "<label>"} <3>
```

Comment:
@adellape based on example
```console
kind: Project
apiVersion: project.openshift.io/v1
metadata:
  name: quay-registry
  annotations:
    openshift.io/node-selector: 'node-role.kubernetes.io/infra='
    scheduler.alpha.kubernetes.io/defaultTolerations: >-
      [{"operator": "Exists", "effect": "NoSchedule", "key":
      "node-role.kubernetes.io/infra"}
      ]
```
the label in <2> is node-role.kubernetes.io/infra=
the label in <3> is node-role.kubernetes.io/infra, no =

but it is same in your description.

could you please check it? thanks

Reply:
I've updated this PR's example `<3>` to be more specific that the user-replaced text should be `<key_name>` in this case, instead of just broadly `<label>`, since this example is using the `Exists` operator, which does not require a value.